rest-service/manager_rest/test/endpoints/test_deployment_labels_dependencies.py

```diff
@@ -1,613 +1,420 @@
-
-from mock import patch
-
 from cloudify_rest_client.exceptions import CloudifyClientError
 from cloudify.models_states import DeploymentState
 
-from manager_rest.rest.rest_utils import RecursiveDeploymentLabelsDependencies
+from manager_rest.storage import models, db
 from manager_rest.test.base_test import BaseServerTestCase
 
 
 class DeploymentLabelsDependenciesTest(BaseServerTestCase):
 
-    def _create_deployment_objects(self, parent_name, deployment_type, size):
-        for service in range(1, size + 1):
-            self.put_deployment_with_labels(
-                [
-                    {
-                        'csys-obj-parent': parent_name
-                    },
-                    {
-                        'csys-obj-type': deployment_type,
-                    }
-                ],
-                resource_id='{0}_{1}_{2}'.format(
-                    deployment_type, service, parent_name)
-            )
-
-    def _populate_deployment_labels_dependencies(self):
-        self.put_mock_deployments('dep_0', 'dep_1')
-        self.put_mock_deployments('dep_2', 'dep_3')
-        self.put_mock_deployments('dep_4', 'dep_5')
-
-        self.client.deployments.update_labels('dep_0', [
-                {
-                    'csys-obj-parent': 'dep_1'
-                }
-            ]
-        )
-
-        self.client.deployments.update_labels('dep_2', [
-                {
-                    'csys-obj-parent': 'dep_3'
-                }
-            ]
+    def setUp(self):
+        super().setUp()
+        self.blueprint = models.Blueprint(
+            tenant=self.tenant,
+            creator=self.user,
         )
 
-        self.client.deployments.update_labels('dep_4', [
-                {
-                    'csys-obj-parent': 'dep_5'
-                }
-            ]
-        )
-
-    @patch('manager_rest.resource_manager.ResourceManager'
-           '.handle_deployment_labels_graph')
-    @patch('manager_rest.resource_manager.ResourceManager'
-           '.verify_attaching_deployment_to_parents')
-    def test_deployment_with_empty_labels(self,
-                                          verify_parents_mock,
-                                          handle_labels_graph_mock):
-        self.put_deployment('deployment_with_no_labels')
-        verify_parents_mock.assert_not_called()
-        handle_labels_graph_mock.assert_not_called()
-
-    @patch('manager_rest.resource_manager.ResourceManager'
-           '.handle_deployment_labels_graph')
-    @patch('manager_rest.resource_manager.ResourceManager'
-           '.verify_attaching_deployment_to_parents')
-    def test_deployment_with_non_parent_labels(self,
-                                               verify_parents_mock,
-                                               handle_labels_graph_mock):
-        self.put_deployment_with_labels([{'env': 'aws'}, {'arch': 'k8s'}])
-        verify_parents_mock.assert_not_called()
-        handle_labels_graph_mock.assert_not_called()
```

Comment:
removing those because they're tightly coupled to methods that I'm going to be removing in RD-3573

Like, no need for a test that only checks that something _wasn't_ called.